testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CustomCLIExecutor.java

```diff
@@ -195,37 +203,6 @@ public static String execute(File cliConfigFile, String operation, String contro
         return exitCode + ": " + cliOutput;
     }
 
-    /**
-     * Waits for server to reload until server-state is running
-     *
-     * @param timeout
-     * @param controller
-     * @throws Exception
-     */
-    public static void waitForServerToReload(int timeout, String controller) throws Exception {
-
-        Thread.sleep(TimeoutUtil.adjust(500));
-        long start = System.currentTimeMillis();
-        long now;
-        do {
-            try {
-                String result = CustomCLIExecutor.execute(null, READ_ATTRIBUTE_OPERATION + " server-state", controller);
-                boolean normal = result.contains("running");
-                if (normal) {
-                    return;
-                }
-            } catch (Exception e) {
-            }
-            try {
-                Thread.sleep(100);
-            } catch (InterruptedException e) {
-            }
-            now = System.currentTimeMillis();
-        } while (now - start < timeout);
-
-        fail("Server did not reload in the imparted time.");
-    }
-
```

Comment:
This is unused method, it was in original commit on master. Just took whole commit...
I can remove it if you want.